[BUG] Stopping Tor doesn't work

**Describe the Bug**

**To Reproduce**
Steps to reproduce the behavior:
1. Click stop Tor
2. Leave app
3. Return
4. It never stopped (notification never went away either)

**Expected Behavior**
Tor stops

**What Custom Configuration Do You Use?**
- Power user mode
- Screen security
- no root detection
- isolate destination adddresses
- isolate destination ports
- isolate client protocols
- prefer ipv6 connections
- connection padding
- circuit padding
- tor transproxy port: auto
- tor dns port: auto
- `HardwareAccel 1`
- `NoExec 1`
- `ClientOnly 1`
- `Sandbox 1`
- `SocksPort [::]:9050 IPv^Traffic PreferIPv6 IsolateDestAddr IsolateDestPort IsolateClientProtocol`
- `HTTPTunnelPort [::]:8118 IPv7Traffic PreferIPv6 IsolateDestAddr IsolateDestPort IsolateClientProtocol`
- no network auto-sleep

**Screenshots**
N/A

**Smartphone (please complete the following information):**
- Device:
- OS: GrapheneOS
- Version: v17.7.0 RC 3

**Crash Logs (Advanced)**
N/A

**Additional Context**
Started with v17.7.0 RC 3

Here is a logcat using the latest on Master branch (de70525a34ed71bf2e1a7790ec18e60a37bbe36b):

```
12-18 20:49:41.723   931   943 I ActivityManager: Start proc 1476:org.torproject.android:tor/u0a322 for service org.torproject.android/.service.OrbotService
12-18 20:50:21.517  1476  1515 D Orbot   : Updating torrc custom configuration…
12-18 20:50:21.524  1476  1515 D Orbot   : torrc.custom=
12-18 20:50:21.524  1476  1515 D Orbot   : RunAsDaemon 1
12-18 20:50:21.524  1476  1515 D Orbot   : AvoidDiskWrites 1
12-18 20:50:21.524  1476  1515 D Orbot   : SOCKSPort 9050 IPv6Traffic PreferIPv6 
12-18 20:50:21.524  1476  1515 D Orbot   : SafeSocks 0
12-18 20:50:21.524  1476  1515 D Orbot   : TestSocks 0
12-18 20:50:21.524  1476  1515 D Orbot   : HTTPTunnelPort 8118 
12-18 20:50:21.524  1476  1515 D Orbot   : ReducedConnectionPadding 1
12-18 20:50:21.524  1476  1515 D Orbot   : CircuitPadding 1
12-18 20:50:21.524  1476  1515 D Orbot   : ReducedCircuitPadding 1
12-18 20:50:21.524  1476  1515 D Orbot   : TransPort 9040 
12-18 20:50:21.524  1476  1515 D Orbot   : DNSPort 9053 
12-18 20:50:21.524  1476  1515 D Orbot   : VirtualAddrNetwork 10.192.0.0/10
12-18 20:50:21.524  1476  1515 D Orbot   : AutomapHostsOnResolve 1
12-18 20:50:21.524  1476  1515 D Orbot   : DormantClientTimeout 10 minutes
12-18 20:50:21.524  1476  1515 D Orbot   : DormantCanceledByStartup 1
12-18 20:50:21.524  1476  1515 D Orbot   : DisableNetwork 0
12-18 20:50:21.524  1476  1515 D Orbot   : UseBridges 0
12-18 20:50:21.524  1476  1515 D Orbot   : GeoIPFile /data/data/org.torproject.android/files/geoip
12-18 20:50:21.524  1476  1515 D Orbot   : GeoIPv6File /data/data/org.torproject.android/files/geoip6
12-18 20:50:21.524  1476  1515 D Orbot   : StrictNodes 0
12-18 20:50:21.536  1476  1515 D Orbot   : binding tor service
12-18 20:50:21.544  1476  1476 I TorService: Acquired lock
12-18 20:50:21.576  1476  1564 D Orbot   : Updating torrc custom configuration…
12-18 20:50:21.580  1476  1564 D Orbot   : torrc.custom=
12-18 20:50:21.580  1476  1564 D Orbot   : RunAsDaemon 1
12-18 20:50:21.580  1476  1564 D Orbot   : AvoidDiskWrites 1
12-18 20:50:21.580  1476  1564 D Orbot   : SOCKSPort 9050 IPv6Traffic PreferIPv6 
12-18 20:50:21.580  1476  1564 D Orbot   : SafeSocks 0
12-18 20:50:21.580  1476  1564 D Orbot   : TestSocks 0
12-18 20:50:21.580  1476  1564 D Orbot   : HTTPTunnelPort 8118 
12-18 20:50:21.580  1476  1564 D Orbot   : ReducedConnectionPadding 1
12-18 20:50:21.580  1476  1564 D Orbot   : CircuitPadding 1
12-18 20:50:21.580  1476  1564 D Orbot   : ReducedCircuitPadding 1
12-18 20:50:21.580  1476  1564 D Orbot   : TransPort 9040 
12-18 20:50:21.580  1476  1564 D Orbot   : DNSPort 9053 
12-18 20:50:21.580  1476  1564 D Orbot   : VirtualAddrNetwork 10.192.0.0/10
12-18 20:50:21.580  1476  1564 D Orbot   : AutomapHostsOnResolve 1
12-18 20:50:21.580  1476  1564 D Orbot   : DormantClientTimeout 10 minutes
12-18 20:50:21.580  1476  1564 D Orbot   : DormantCanceledByStartup 1
12-18 20:50:21.580  1476  1564 D Orbot   : DisableNetwork 0
12-18 20:50:21.580  1476  1564 D Orbot   : UseBridges 0
12-18 20:50:21.580  1476  1564 D Orbot   : GeoIPFile /data/data/org.torproject.android/files/geoip
12-18 20:50:21.580  1476  1564 D Orbot   : GeoIPv6File /data/data/org.torproject.android/files/geoip6
12-18 20:50:21.580  1476  1564 D Orbot   : StrictNodes 0
12-18 20:50:21.587  1476  1564 D Orbot   : binding tor service
12-18 20:50:26.369  1476  1476 W art     : Long monitor contention with owner ControlSocket (1567) at void java.lang.Object.wait!()(Object.java:-2) waiters=0 in java.util.List net.freehaven.tor.control.TorControlConnection.sendAndWaitForResponse(java.lang.String, java.lang.String) for 3.317s
12-18 20:50:26.376  1476  1476 D Orbot   : Connected to tor control port
12-18 20:50:26.656  1476  1569 D Orbot   : orConnStatus (setsun): LAUNCHED
12-18 20:50:26.656  1476  1569 D Orbot   : NOTICE: Bootstrapped 5% (conn): Connecting to a relay
12-18 20:50:26.678  1476  1569 D Orbot   : orConnStatus (McAdam): LAUNCHED
12-18 20:50:26.705  1476  1569 D Orbot   : NOTICE: Bootstrapped 10% (conn_done): Connected to a relay
12-18 20:50:26.709  1476  1569 D Orbot   : NOTICE: Bootstrapped 14% (handshake): Handshaking with a relay
12-18 20:50:26.770  1476  1569 D Orbot   : NOTICE: Bootstrapped 15% (handshake_done): Handshake with a relay done
12-18 20:50:26.774  1476  1569 D Orbot   : NOTICE: Bootstrapped 75% (enough_dirinfo): Loaded enough directory info to build circuits
12-18 20:50:26.783  1476  1569 D Orbot   : NOTICE: Bootstrapped 90% (ap_handshake_done): Handshake finished with a relay to build circuits
12-18 20:50:26.791  1476  1569 D Orbot   : NOTICE: Bootstrapped 95% (circuit_create): Establishing a Tor circuit
12-18 20:50:26.796  1476  1569 D Orbot   : orConnStatus (setsun): CONNECTED
12-18 20:50:26.913  1476  1569 D Orbot   : NOTICE: Bootstrapped 100% (done): Done
12-18 20:50:27.141  1476  1569 D Orbot   : orConnStatus (McAdam): CONNECTED
12-18 20:50:27.669  1476  1476 D Orbot   : Connected to tor control port
12-18 20:50:38.750  1476  1564 D OrbotVpnManager: stopping VPN
12-18 20:50:38.750  1476  1564 D Orbot   : stopTorAsync
12-18 20:50:38.762  1476  1564 D Orbot   : unbinding tor service
12-18 20:50:48.975  1476  1564 D Orbot   : Updating torrc custom configuration…
12-18 20:50:48.980  1476  1564 D Orbot   : torrc.custom=
12-18 20:50:48.980  1476  1564 D Orbot   : RunAsDaemon 1
12-18 20:50:48.980  1476  1564 D Orbot   : AvoidDiskWrites 1
12-18 20:50:48.980  1476  1564 D Orbot   : SOCKSPort 9050 IPv6Traffic PreferIPv6 
12-18 20:50:48.980  1476  1564 D Orbot   : SafeSocks 0
12-18 20:50:48.980  1476  1564 D Orbot   : TestSocks 0
12-18 20:50:48.980  1476  1564 D Orbot   : HTTPTunnelPort 8118 
12-18 20:50:48.980  1476  1564 D Orbot   : ReducedConnectionPadding 1
12-18 20:50:48.980  1476  1564 D Orbot   : CircuitPadding 1
12-18 20:50:48.980  1476  1564 D Orbot   : ReducedCircuitPadding 1
12-18 20:50:48.980  1476  1564 D Orbot   : TransPort 9040 
12-18 20:50:48.980  1476  1564 D Orbot   : DNSPort 9053 
12-18 20:50:48.980  1476  1564 D Orbot   : VirtualAddrNetwork 10.192.0.0/10
12-18 20:50:48.980  1476  1564 D Orbot   : AutomapHostsOnResolve 1
12-18 20:50:48.980  1476  1564 D Orbot   : DormantClientTimeout 10 minutes
12-18 20:50:48.980  1476  1564 D Orbot   : DormantCanceledByStartup 1
12-18 20:50:48.980  1476  1564 D Orbot   : DisableNetwork 0
12-18 20:50:48.980  1476  1564 D Orbot   : UseBridges 0
12-18 20:50:48.980  1476  1564 D Orbot   : GeoIPFile /data/data/org.torproject.android/files/geoip
12-18 20:50:48.980  1476  1564 D Orbot   : GeoIPv6File /data/data/org.torproject.android/files/geoip6
12-18 20:50:48.980  1476  1564 D Orbot   : StrictNodes 0
12-18 20:50:48.994  1476  1564 D Orbot   : binding tor service
12-18 20:50:49.036  1476  1515 D Orbot   : Updating torrc custom configuration…
12-18 20:50:49.039  1476  1515 D Orbot   : torrc.custom=
12-18 20:50:49.039  1476  1515 D Orbot   : RunAsDaemon 1
12-18 20:50:49.039  1476  1515 D Orbot   : AvoidDiskWrites 1
12-18 20:50:49.039  1476  1515 D Orbot   : SOCKSPort 9050 IPv6Traffic PreferIPv6 
12-18 20:50:49.039  1476  1515 D Orbot   : SafeSocks 0
12-18 20:50:49.039  1476  1515 D Orbot   : TestSocks 0
12-18 20:50:49.039  1476  1515 D Orbot   : HTTPTunnelPort 8118 
12-18 20:50:49.039  1476  1515 D Orbot   : ReducedConnectionPadding 1
12-18 20:50:49.039  1476  1515 D Orbot   : CircuitPadding 1
12-18 20:50:49.039  1476  1515 D Orbot   : ReducedCircuitPadding 1
12-18 20:50:49.039  1476  1515 D Orbot   : TransPort 9040 
12-18 20:50:49.039  1476  1515 D Orbot   : DNSPort 9053 
12-18 20:50:49.039  1476  1515 D Orbot   : VirtualAddrNetwork 10.192.0.0/10
12-18 20:50:49.039  1476  1515 D Orbot   : AutomapHostsOnResolve 1
12-18 20:50:49.039  1476  1515 D Orbot   : DormantClientTimeout 10 minutes
12-18 20:50:49.039  1476  1515 D Orbot   : DormantCanceledByStartup 1
12-18 20:50:49.039  1476  1515 D Orbot   : DisableNetwork 0
12-18 20:50:49.039  1476  1515 D Orbot   : UseBridges 0
12-18 20:50:49.039  1476  1515 D Orbot   : GeoIPFile /data/data/org.torproject.android/files/geoip
12-18 20:50:49.039  1476  1515 D Orbot   : GeoIPv6File /data/data/org.torproject.android/files/geoip6
12-18 20:50:49.039  1476  1515 D Orbot   : StrictNodes 0
12-18 20:50:49.046  1476  1515 D Orbot   : binding tor service
12-18 20:50:50.002  1476  1476 E Orbot   : java.io.IOException: Tor is no longer running
12-18 20:50:50.002  1476  1476 D Orbot   : stopTorAsync
12-18 20:50:50.008  1476  1476 D Orbot   : error shutting down Tor from the control port
12-18 20:50:50.008  1476  1476 D Orbot   : unbinding tor service
12-18 20:50:50.057  1476  1476 E ActivityThread: Service org.torproject.android.service.OrbotService has leaked ServiceConnection org.torproject.android.service.OrbotService$1@b78efa that was originally bound here
12-18 20:50:50.057  1476  1476 E ActivityThread: android.app.ServiceConnectionLeaked: Service org.torproject.android.service.OrbotService has leaked ServiceConnection org.torproject.android.service.OrbotService$1@b78efa that was originally bound here
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at android.app.LoadedApk$ServiceDispatcher.<init>(LoadedApk.java:1336)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at android.app.LoadedApk.getServiceDispatcher(LoadedApk.java:1231)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1450)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at android.app.ContextImpl.bindService(ContextImpl.java:1422)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at android.content.ContextWrapper.bindService(ContextWrapper.java:636)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.startTorService(OrbotService.java:492)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.lambda$startTor$0(OrbotService.java:365)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.$r8$lambda$uY3IB_AuX9kepReSzgSujwykO3Y(OrbotService.java)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService$$ExternalSyntheticLambda0.invoke(D8$$SyntheticClass:0)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.circumvention.SmartConnect.handle(SmartConnect.kt:32)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.startTor(OrbotService.java:363)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.-$$Nest$mstartTor(OrbotService.java)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService$IncomingIntentRouter.run(OrbotService.java:691)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
12-18 20:50:50.057  1476  1476 E ActivityThread: 	at java.lang.Thread.run(Thread.java:761)
12-18 20:50:50.059  1476  1476 E ActivityThread: Service org.torproject.android.service.OrbotService has leaked ServiceConnection org.torproject.android.service.OrbotService$1@3b541ee that was originally bound here
12-18 20:50:50.059  1476  1476 E ActivityThread: android.app.ServiceConnectionLeaked: Service org.torproject.android.service.OrbotService has leaked ServiceConnection org.torproject.android.service.OrbotService$1@3b541ee that was originally bound here
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at android.app.LoadedApk$ServiceDispatcher.<init>(LoadedApk.java:1336)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at android.app.LoadedApk.getServiceDispatcher(LoadedApk.java:1231)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1450)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at android.app.ContextImpl.bindService(ContextImpl.java:1422)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at android.content.ContextWrapper.bindService(ContextWrapper.java:636)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.startTorService(OrbotService.java:492)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.lambda$startTor$0(OrbotService.java:365)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.$r8$lambda$uY3IB_AuX9kepReSzgSujwykO3Y(OrbotService.java)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService$$ExternalSyntheticLambda0.invoke(D8$$SyntheticClass:0)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.circumvention.SmartConnect.handle(SmartConnect.kt:32)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.startTor(OrbotService.java:363)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService.-$$Nest$mstartTor(OrbotService.java)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at org.torproject.android.service.OrbotService$IncomingIntentRouter.run(OrbotService.java:691)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
12-18 20:50:50.059  1476  1476 E ActivityThread: 	at java.lang.Thread.run(Thread.java:761)
12-18 20:50:50.063  1476  1476 I TorService: Releasing lock
12-18 20:50:50.064  1476  1476 W System.err: java.io.IOException: write failed: EPIPE (Broken pipe)
12-18 20:50:50.067  1476  1476 W System.err: 	at libcore.io.IoBridge.write(IoBridge.java:501)
12-18 20:50:50.067  1476  1476 W System.err: 	at java.io.FileOutputStream.write(FileOutputStream.java:316)
12-18 20:50:50.067  1476  1476 W System.err: 	at sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:221)
12-18 20:50:50.067  1476  1476 W System.err: 	at sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:291)
12-18 20:50:50.067  1476  1476 W System.err: 	at sun.nio.cs.StreamEncoder.implFlush(StreamEncoder.java:295)
12-18 20:50:50.067  1476  1476 W System.err: 	at sun.nio.cs.StreamEncoder.flush(StreamEncoder.java:141)
12-18 20:50:50.067  1476  1476 W System.err: 	at java.io.OutputStreamWriter.flush(OutputStreamWriter.java:229)
12-18 20:50:50.067  1476  1476 W System.err: 	at net.freehaven.tor.control.TorControlConnection.shutdownTor(TorControlConnection.java:676)
12-18 20:50:50.067  1476  1476 W System.err: 	at org.torproject.jni.TorService.shutdownTor(TorService.java:496)
12-18 20:50:50.067  1476  1476 W System.err: 	at org.torproject.jni.TorService.onDestroy(TorService.java:462)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.app.ActivityThread.handleStopService(ActivityThread.java:3381)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.app.ActivityThread.-wrap27(ActivityThread.java)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1606)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:102)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.os.Looper.loop(Looper.java:154)
12-18 20:50:50.067  1476  1476 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
12-18 20:50:50.067  1476  1476 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
12-18 20:50:50.067  1476  1476 W System.err: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
12-18 20:50:50.067  1476  1476 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
12-18 20:50:50.069  1476  1476 W System.err: Caused by: android.system.ErrnoException: write failed: EPIPE (Broken pipe)
12-18 20:50:50.069  1476  1476 W System.err: 	at libcore.io.Posix.writeBytes(Native Method)
12-18 20:50:50.069  1476  1476 W System.err: 	at libcore.io.Posix.write(Posix.java:273)
12-18 20:50:50.070  1476  1476 W System.err: 	at libcore.io.BlockGuardOs.write(BlockGuardOs.java:319)
12-18 20:50:50.070  1476  1476 W System.err: 	at libcore.io.IoBridge.write(IoBridge.java:496)
12-18 20:50:50.070  1476  1476 W System.err: 	... 18 more
12-18 20:51:29.121  1476  1839 D OrbotVpnManager: stopping VPN
12-18 20:51:29.122  1476  1839 D Orbot   : stopTorAsync
```

--- a/app/src/main/java/org/torproject/android/ui/connect/ConnectFragment.kt
+++ b/app/src/main/java/org/torproject/android/ui/connect/ConnectFragment.kt
@@ -168,7 +168,6 @@ class ConnectFragment : Fragment(),
                     with(binding.btnStart) {
                         text = context.getString(android.R.string.cancel)
                     }
-                    requireContext().sendIntentToService(OrbotConstants.ACTION_START)
                 }
             }
             doLayoutStarting(requireContext())

